geometry_msgs/Pose part_pose
---
int8 num_sols
ik_service/JointSolutions[8] joint_solutions
